app/controllers/api/v1/merchants/find_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::V1::Merchants::FindController < ApplicationController
+
+  def show
```

Comment:
@icorson3 we are wondering if this is the best way to go about this - we namespaced this find path under merchants, and put a show method there that returns what we want based on params